LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -248,19 +300,22 @@ function addHeader(column) {
             });
             break;
 	    default:
-	    	if(column.substr(0, 6) == 'depID_') { // backwards compatibility for LEAF workflow requirement based approval dates
-	    		depID = column.substr(6);
```

Comment:
Was this not working before?

Reply:
Not sure which part you have a question about.  If it's the addition of "let" in front of the variable, it's to properly scope it.  Looks like it might have been depending on it being instantiated else where globally.  Always and iffy prospect.

Reply:
That one does not appear to be used outside of that block.  However, I've also discussed with Pete and Greg and we all agree that it's better not to touch anything that could affect a workflow unless we are 100% sure on it, so I was going to revert this change unless that's objected to.